csc207/src/main/java/use_case/add_expense/AddExpenseInputData.java

```diff
@@ -7,18 +7,16 @@
  */
 public class AddExpenseInputData {
     private final String name;
-    private final double amount;
+    private final String amount;
     private final String category;
-    private final LocalDate date;
 
-    public AddExpenseInputData(String name, double amount, String category, LocalDate date) {
+    public AddExpenseInputData(String name, String amount, String category) {
```

Comment:
why do we use this? it seems like it's nearly the same content as a Transaction/Expense?